gulpfile.js

```diff
@@ -32,12 +32,17 @@ gulp.task('test', function (cb) {
     make('testLegacy', cb);
 });
 
+gulp.task('layout', function (cb) {
```

Comment:
the purpose of the this file (gulpfile.js) is to maintain compat with the CI definitions. the build process in newer branches follows the "make-pattern". but in case m105 or 2015.4 needs to be built, we have refrained from editing the build definition.

given that this file is for compat, i dont think it makes sense to add a new gulp task. the CI calls `gulp && gulp test`. the publish CI only calls `gulp publish`. so i think the modifications should only be made to the gulp "build" task.